hello-app/cloudshell/tutorial.md

```diff
@@ -0,0 +1,133 @@
+# Hello Application example
+
+## 
+
+This tutorial shows how to build and deploy a containerized Go web server application using [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine).
+
+You'll learn how to create a cluster, and how to deploy the application to the cluster so that it can be accessed by users.
+
+Let's get started!
+
+
+## Set your GCP project
+
+You can use an existing Google Cloud Platform project for this tutorial, or you can [create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
+
+Open the <walkthrough-editor-spotlight spotlightId="menu-terminal">terminal</walkthrough-editor-spotlight> and follow the steps below:
+
+### 1. Set the default project    
+```bash
+gcloud config set project PROJECT_ID
+```     
+Replace PROJECT_ID with your [project id](https://support.google.com/cloud/answer/6158840).
+
+### 2. Set the default compute/zone    
+```bash
+gcloud config set compute/zone COMPUTE_ZONE
+```    
+Replace COMPUTE_ZONE with your [compute zone](https://cloud.google.com/compute/docs/regions-zones#available), such as `us-west1-a`.
+
+
+## Create a GKE cluster
+A cluster consists of at least one cluster control plane machine and multiple worker machines called nodes. Nodes are [Compute Engine virtual machine (VM) instances](https://cloud.google.com/compute/docs/instances) that run the Kubernetes processes necessary to make them part of the cluster. 
+
+GKE offers two [modes of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters#modes) for clusters: Standard and Autopilot. For this tutorial, we'll use Standard mode.
+
+### 1. Create a Standard GKE cluster
+
+Run the command below in your terminal to create a one-node Standard cluster named `hello-cluster`:
+```
+gcloud container clusters create hello-cluster --num-nodes=1
+```
+
+It might take several minutes to finish creating the cluster.
+
+
+### 2. Get authentication credentials
+
+After creating your cluster, you need to get authentication credentials to interact with the cluster.
+
+```
+gcloud container clusters get-credentials hello-cluster
+```
+
+This command configures `kubectl` to use the cluster you created.    
+
+
+Next, let's deploy an app to the cluster.
+
+## Deploy an application to the cluster
+
+Now that you have created a cluster, you can deploy a [containerized application](https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview#workloads) to it. For this tutorial, let's deploy the example web application `hello-app`.
```

Comment:
```suggestion
Now that you have created a cluster, you can deploy a [containerized application](https://cloud.google.com/kubernetes-engine/docs/concepts/kubernetes-engine-overview#workloads) to it. For this tutorial, let's deploy the [example web application](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/tree/master/hello-app) `hello-app`.
```

let's link back to the github repo? If I stubble across this tutorial from somewhere else, I would like to know what I'm deploying